guides/source/active_job_basics.md

```diff
@@ -406,6 +410,72 @@ end
 [`around_perform`]: https://api.rubyonrails.org/classes/ActiveJob/Callbacks/ClassMethods.html#method-i-around_perform
 [`after_perform`]: https://api.rubyonrails.org/classes/ActiveJob/Callbacks/ClassMethods.html#method-i-after_perform
 
+Please note that when enqueueing jobs in bulk using `perform_all_later`, callbacks such as `around_enqueue`, will not be triggered on the individual jobs. See [Bulk Enqueuing Callbacks](#bulk-enqueue-callbacks).
```

Comment:
> (...) callbacks such as `around_enqueue` will not (...)

"Callbacks" is the subject of the phrase and "will not be triggered" is the verb in passive voice mode. We may not separate the verb from its subject with a comma.